Code/api/models/projects.js

```diff
@@ -0,0 +1,24 @@
+var mongoose    = require('mongoose');
+var Schema      = mongoose.Schema;
+
+var ProjectSchema = new Schema({
+    owner: String,
+    title: String,
+    description: String,
+    disciplines: [String],
```

Comment:
I'm not 100% on this line...to declare an array, you just do [], then fill in what you need inside.